### PR TITLE
Features perf mark

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**

### CODE REVIEW

**Review by alxhub — APPROVED** [review]
LGTM. Should we make constants for the objects here, to avoid the allocations?

**Comment by pkozlowski-opensource** [comment]
> Should we make constants for the objects here, to avoid the allocations?

Oh, consts are definitively a good call! Changed!

File: packages/core/src/render3/instructions/control_flow.ts [hunk 1]
```diff
@@ -36,6 +40,8 @@ import {ɵɵtemplate} from './template';
  * @codeGenApi
  */
 export function ɵɵconditional<T>(containerIndex: number, matchingTemplateIndex: number, value?: T) {
+  performance.mark('mark_use_counter', PERF_MARK_CONTROL_FLOW);
```
[the remark this answers is not in the document]
Comment (pkozlowski-opensource) [reply packages/core/src/render3/instructions/control_flow.ts:43]: We've discussed this extensively with the Chrome team and calling this API, even very often, should be close to noop from the perf point of view.

**Review by AndrewKushnir — APPROVED** [review]

**Comment by dylhunn** [comment]
@pkozlowski-opensource Looks like this needs a local mod in g3 to be updated.

**Comment by pkozlowski-opensource** [comment]
Thnx for the info @dylhunn - I've removed the problematic change from this PR and will do it in a separate PR. This one is ready to merge now.

**Comment by alxhub** [comment]
This PR was merged into the repository by commit 225914b75ef038a6e97885b87571e90a82e9e181.

**Pull request closed without merging**